graphql2/graphqlapp/dataloaders.go

```diff
@@ -37,9 +39,23 @@ func (a *App) registerLoaders(ctx context.Context) context.Context {
 	ctx = context.WithValue(ctx, dataLoaderKeyService, dataloader.NewServiceLoader(ctx, a.ServiceStore))
 	ctx = context.WithValue(ctx, dataLoaderKeyUser, dataloader.NewUserLoader(ctx, a.UserStore))
 	ctx = context.WithValue(ctx, dataLoaderKeyCM, dataloader.NewCMLoader(ctx, a.CMStore))
+	ctx = context.WithValue(ctx, dataLoaderKeyAlertLogMessageStatus, dataloader.NewAlertLogMessageStatusLoader(ctx, a.NotificationStore))
 	return ctx
 }
 
+func (app *App) FindOneAlertLogMessageStatus(ctx context.Context, id string) (*notification.MessageStatus, error) {
```

Comment:
We should probably update the names to be `NotificationMessageStatus` for consistency